README.md

```diff
@@ -1,19 +1,18 @@
 # gluon-mesh-vpn-wireguard
 
-You can use this package for connecting with wireguard to the freifunk Königswinter network.
+You can use this package for connecting with wireguard to the Freifunk Munich network.
 
 You should use something like the following in the site.conf:
 
         
 ```
  mesh_vpn = {
-        mtu = 1420,
+        mtu = 1410,
```

Comment:
What are you aligning your MTU against?

I see 80 Bytes (Wireguard) + 8 Byte (VXLAN), so it looks like you are aiming to support a lower link that supports 1498 Bytes? That seems insanely large, when DSL is usually 1492 Bytes (Ethernet - PPP) and cable links out there are doing the weirdest shit.

Reply:
This was just copy&paste from another setup and you are right that‘s nonsense :).